### PR TITLE
Proper markdown support on descriptions

### PR DESCRIPTION
**Pull request by ildyria**
Closes #2334

Makes use of https://github.com/tailwindlabs/tailwindcss-typography

### CODE REVIEW

File: resources/views/components/gallery/photo/sidebar.blade.php [hunk 1]
```diff
@@ -17,8 +17,7 @@ class="border-t border-solid border-primary-500 text-text-main-0 w-[360px] h-ful
             <span class="col-span-2 py-0.5 pl-3 text-sm">{{ __('lychee.PHOTO_DESCRIPTION') }}</span>
         </template>
         <template x-if="preformatted.description">
-            <div class="pb-0.5 pt-4 pl-8 col-span-2 text-sm" id="photoDescription"
-                x-html="preformatted.description">
+            <div class="pb-0.5 pt-4 pl-8 col-span-2 prose prose-invert prose-sm" x-html="preformatted.description">
```
Comment (d7415) [inline resources/views/components/gallery/photo/sidebar.blade.php:20]: We don't use the ID?

**Review by d7415 — APPROVED** [review]
Untested. One query, otherwise no concerns.

File: resources/views/components/gallery/photo/sidebar.blade.php (hunk 1 above)
Comment (ildyria) [reply resources/views/components/gallery/photo/sidebar.blade.php:20]: We used to for the css, but it is no longer required with the `prose`.

Comment (ildyria) [reply resources/views/components/gallery/photo/sidebar.blade.php:20]: See https://github.com/LycheeOrg/Lychee/pull/2335/files#diff-970bcdf1edf2cc8409fb3fca71f2fe90be7151004bbad5febe75056386205668

**Pull request merged**